frontend/src/app/commonComponents/Required.tsx

```diff
@@ -3,8 +3,9 @@ import React from "react";
 const Required = (props: { label?: React.ReactNode }) => (
   <>
     {props.label}
```

Comment:
How do I add a space only when there's a label? e.g. "` *`" so `My label *`. And no space when there's no label, e.g. `(*)`

Reply:
Done in feabca7